src/zenml/integrations/airflow/__init__.py

```diff
@@ -32,7 +32,14 @@ class AirflowIntegration(Integration):
     NAME = AIRFLOW
     # remove pendulum version requirement once Airflow supports
     # pendulum>-3.0.0
-    REQUIREMENTS = ["apache-airflow~=2.4.0", "pendulum<3.0.0"]
+    REQUIREMENTS = [
+        "apache-airflow~=2.4.0",
+        "pendulum<3.0.0",
+        # We need to add this as an extra dependency to manually downgrade
+        # SQLModel. Otherwise, the initial installation of ZenML installs
+        # a higher version SQLModel and a version mismatch is created.
+        "sqlmodel>=0.0.9,<=0.0.16",
```

Comment:
Is this still needed? You have exactly the same in `pyproject.toml` now...

Reply:
Yes, if we don't put this here, when someone installs ZenML, it installs sqlmodel 0.0.16. When they install the airflow integration on top, it downgrades sqlalchemy but not the sqlmodel, which creates a drift.